tools/dockerversioning/vendor/gopkg.in/yaml.v2/.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: go
```

Comment:
Please, don't commit /vendor folder. 

Not checking anywhere here: https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/versioning/

And add the folder to gitignore

Reply:
Discussed directly - we decided to  commit our vendor folder.